frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -279,7 +279,14 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
               icon={getMarkerIcon(cluster)}
               onclick={() => {
                 onMarkerClick(); //open information slideout
-                popUpContext.setPropertyInfo(convertToProperty(cluster.properties));
+                const [longitude, latitude] = cluster.geometry.coordinates;
+                popUpContext.setPropertyInfo(
+                  convertToProperty({
+                    ...cluster.properties,
+                    latitude: latitude,
```

Comment:
the cluster doesn't contain the lat/lng which is required for the view/update/zoom actions in the slide out.

Reply:
huh, weird that it wasn't giving me any issues before